fh-android-sdk/src/main/java/com/feedhenry/sdk/AppProps.java

```diff
@@ -0,0 +1,159 @@
+package com.feedhenry.sdk;
```

Comment:
In general in this file I see a lot of code of the format 

```java
/** Returns foo
* @returns foo
/*
public Foo getFoo() {
  return foo;
}
```

It would be really nice if we had a quick description about what foo was (or a link to online documentation for foo)